openday_scavenger/views/admin/templates/puzzles.html

```diff
@@ -24,53 +34,98 @@ <h5 class="border-bottom pb-2 mb-0">Available Puzzles</h5>
 <!-- Modal -->
 <div class="modal fade modal-lg" id="addPuzzleModal" tabindex="-1">
     <div class="modal-dialog modal-dialog-centered">
-      <div class="modal-content">
-        <div class="modal-header">
-          <h1 class="modal-title fs-5">Add a new Puzzle to the Scavenger Hunt</h1>
-          <button type="button" class="btn-close" data-bs-dismiss="modal"></button>
-        </div>
-        <form id='puzzle-add-form' class="mt-3" hx-encoding='multipart/form-data' hx-post='/admin/puzzles' hx-ext='json-enc' hx-swap="innerHTML" hx-target="#puzzle-table">
-            <div class="modal-body">
-                <div class="container">
-                    <div class="row">
-                        <div class="col">
-                            <div class="mb-3">
-                                <label for="name" class="form-label">Name:</label>
-                                <input type="text" id="name" name="name" class="form-control">
+        <div class="modal-content">
```

Comment:
Oops, there were some indentation issues here so I reformatted the file. You might need to "Hide whitespace" in Github so you can actually review it.